chapters/fundamentals.adoc

```diff
@@ -1255,7 +1255,7 @@ ifdef::VK_VERSION_1_1,VK_KHR_get_physical_device_properties2[]
 Physical-device-level functionality or behavior added by a <<extensions,
 device extension>> to the API must: not be used unless the conditions
 described in <<initialization-phys-dev-extensions, Extending Physical Device
-Core Functionality>> are met.
+From Device Extensions>> are met.
```

Comment:
I agree that this one is a simple typo and we should accept this part of the change.